backend/tracim_backend/tests/utils.py

```diff
@@ -55,6 +56,20 @@ def get(
         )
 
 
+class ShareApiFactory(object):
+    def __init__(self, session: Session, app_config: CFG, admin_user: User):
+        self.session = session
+        self.app_config = app_config
+        self.admin_user = admin_user
+
+    def get(self, current_user: typing.Optional[User] = None) -> ShareLib:
```

Comment:
`create` is more appropriate for creation. `get` is more oriented "return something already existing".

Reply:
we can change this but get is method used for other factories. Changing all of them is a big refactor (but maybe not too difficult), changing only one make thing inconsistent.

Reply:
get and create usage is clear for normal object but here idea behind this is more "i want a valid lib, with this configuration, i don't mind about which is done behind".

Reply:
see this later

Reply:
https://github.com/tracim/tracim/issues/2169